src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxNumber = 100;

```

Comment:
The 'maxNumber' should be declared as 'static' since it's a constant value that does not change per instance of the class.